frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -63,3 +64,21 @@ const pruneRecursive = (current) => {
 export const prune = (obj) => {
   return pruneRecursive(_.cloneDeep(obj));
 };
+
+// Safely parse js obj to yaml. Returns fallback (emtpy string by default) on exception.
```

Comment:
Want to put this in `src/utils/yaml.js`?

Reply:
Yeah, we can do that

Reply:
I think I might make these return Promises as well. Working on another bug now that touches some of this and realizing that Promises would be a better solution.